pulumi/src/cluster/docker.ts

```diff
@@ -0,0 +1,64 @@
+import { hashElement } from "folder-hash"
+import { Dockerhub } from ".."
+import { hasTag, buildAndPushImage } from "../docker"
+import { getBaseHash, getVolumeReaperHJash as getVolumeReaperHash } from "../hasher"
+
+
```

Comment:
this will actually resolve if you want to run:
```bash
prettier -w --config ../../node/.prettierrc --ignore-path ../../node/.prettierignore ../src/
```
from `unchained/pulumi/volumeReaper`